setup.py

```diff
@@ -1,6 +1,6 @@
 from setuptools import setup, find_packages
 
-version = '0.3.2'
+version = '0.3.3'
```

Comment:
maybe this should be 0.4.0 although this change is not backward compatible, but kind of feels like it requires more than a patch version change. just a thought.

Reply:
good point!  i'll update it.